tests/dummy/config/environment.js

```diff
@@ -18,55 +22,29 @@ module.exports = function(environment) {
       // when it is created
     },
 
+    bootstrapVersion: process.env.BOOTSTRAPVERSION || 4,
+    failOnDeprecation: !!process.env.FAIL_ON_DEPRECATION,
     contentSecurityPolicy: {
-      'default-src': ["'none'"],
       'script-src': [
         "'self'",
-        "'sha256-1xtiB6mV1iIKZ5iz9CxA5lEnfEg8d0XEH3FL9L8NBqo='",
-        "use.typekit.net"
-      ],
-      'font-src': [
-        "'self'",
-        "use.typekit.net",
-        "use.fontawesome.com"
-      ],
-      'connect-src': [
-        "'self'",
-        "performance.typekit.net"
+        "'unsafe-eval'"
```

Comment:
Webpack uses `eval()` by default. I tried to workaround this, but it seems I wrongly added the [`forbidEval`](https://github.com/ef4/ember-auto-import#i-use-content-security-policy-csp-and-it-breaks-ember-auto-import) config to the docs app instead of the dummy app...

Fixed that, and removed `'unsafe-eval'`. Hopefully tests will still pass...